packages/server/lambdas/alfresco/list-files.ts

```diff
@@ -71,10 +72,9 @@ const searchByTermWithParent = async (
 
 export const getFolder = async (uid: string, nodeId: string) => {
   try {
-    const alfrescoCoreAPIUrl = `${getAlfrescoUrlBase()}/alfresco/versions/1`;
-    const url = `${alfrescoCoreAPIUrl}/nodes/${nodeId}?where=(isFolder=true)&include=path`;
+    const url = `${alfrescoApiVersion}/nodes/${nodeId}?where=(isFolder=true)&include=path`;
     const options = await getAlfrescoOptions(uid, { 'Content-Type': 'application/json;charset=UTF-8' });
-    const response = await axios.get(url, options);
+    const response = await alfrescoAxios.get(url, options);
     return response.data;
   } catch (error: any) {
     if (error instanceof AxiosError) {
```

Comment:
Could you try finding out why non-existing sub-folder doesn't enter this 404 error handling block? Cause I receive this error from [logs](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logsV2:log-groups/log-group/$252Faws$252Flambda$252Flambda-rataextra-feat-260-alfresco-list-files/log-events/2023$252F06$252F14$252F$255B$2524LATEST$255D9580701bf6994b7ba3360019fbd04b78):
Example request with non-existing sub-folder: http://localhost:3002/api/alfresco/files?category=hallintaraportit&nestedFolderId=123&page=0
```
{
    "level": "error",
    "time": "2023-06-14T15:10:29.375Z",
    "correlationId": "1-6489d865-67acb97576fa06c52dc90e38",
    "tag": "RATAEXTRA_BACKEND",
    "status": 404,
    "message": "Request failed with status code 404",
    "stack": "AxiosError: Request failed with status code 404\n    at settle (file:///var/task/index.mjs:269862:12)\n    at IncomingMessage.handleStreamEnd (file:///var/task/index.mjs:270523:11)\n    at IncomingMessage.emit (node:events:525:35)\n    at IncomingMessage.emit (node:domain:489:12)\n    at endReadableNT (node:internal/streams/readable:1358:12)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)"
}
```
Otherwise, example request with an existing sub-folder is working properly: http://localhost:3002/api/alfresco/files?category=hallintaraportit&nestedFolderId=aa8fffad-7cb1-4d04-8019-b30a5b0fc84e&page=0

Reply:
Hmm, I think axios intercepts its beforehand and returns the simplified axios error, so it never gets there.. 🤔